examples/managementcli/main.go

```diff
@@ -620,6 +647,13 @@ func main() {
 		cmd.Args = cobra.ExactArgs(2)
 	})
 
+	addCommand(authzLoadSchema, "authz-load-schema", "Load and display the current AuthZ ReBAC schema", func(cmd *cobra.Command) {
```

Comment:
out of curiosity - why only those two? (I'm not arguing we should add all of the authz api here, just wondering what is the logic behind those two, if we want to tell the entire story, maybe also expose save schema and add a relation - like you mentioned in the readme)

Reply:
Just me being lazy. I'll add the save schema and add relation as well so you can have the full cycle via CLI.